cmake/sdlplatform.cmake

```diff
@@ -29,6 +29,8 @@ macro(SDL_DetectCMakePlatform)
     elseif(CMAKE_SYSTEM_NAME MATCHES "Minix.*")
       set(SDL_CMAKE_PLATFORM MINIX)
     endif()
+  elseif(CMAKE_SYSTEM_NAME MATCHES "QNX")
```

Comment:
This is still outside the UNIX section.

```
diff --git a/cmake/sdlplatform.cmake b/cmake/sdlplatform.cmake
index 5659dfb30..6d8ece6ac 100644
--- a/cmake/sdlplatform.cmake
+++ b/cmake/sdlplatform.cmake
@@ -28,9 +28,9 @@ macro(SDL_DetectCMakePlatform)
       set(SDL_CMAKE_PLATFORM AIX)
     elseif(CMAKE_SYSTEM_NAME MATCHES "Minix.*")
       set(SDL_CMAKE_PLATFORM MINIX)
+    elseif(CMAKE_SYSTEM_NAME MATCHES "QNX")
+      set(SDL_CMAKE_PLATFORM QNX)
     endif()
-  elseif(CMAKE_SYSTEM_NAME MATCHES "QNX")
-    set(SDL_CMAKE_PLATFORM QNX)
   elseif(APPLE)
     if(CMAKE_SYSTEM_NAME MATCHES ".*Darwin.*")
       set(SDL_CMAKE_PLATFORM DARWIN)
```

Reply:
Oops. Dumb me. Thanks for double checking this.